certified-connectors/Power Textor/README.md

```diff
@@ -0,0 +1,96 @@
+# Power Textor Connector
+
+## Authentication
+
+Power Textor custom connector uses API key authentication. When creating a connection for the Power Textor custom connector, an API key needs to be provided. This API key can be obtained through the Power Textor website, the steps for which have been outlined in the following section.
+
```

Comment:
@MohsinAQ Please refer the template provided for readme.md file and add required sections.